packages/puppeteer-extra-plugin-recaptcha/src/index.ts

```diff
@@ -331,7 +334,7 @@ export class PuppeteerExtraPluginRecaptcha extends PuppeteerExtraPlugin {
     prop.enterRecaptchaSolutions = async (solutions: types.CaptchaSolution[]) =>
       this.enterRecaptchaSolutions(prop, solutions)
     // Add convenience methods that wraps all others
-    prop.solveRecaptchas = async () => this.solveRecaptchas(prop)
+    prop.solveRecaptchas = this.solveRecaptchas.bind(this, prop)
```

Comment:
i dont know if this syntax works with `async`. `bind()` here makes sure that the first parameter is bound to the page object. Further calls to `prop.solveRecaptchas` should then call the `solveRecaptchas` with just 1 argument (the `extraData`) since the `page` has been bound thus disappeared

Reply:
tested this locally; works correctly (and properly assigns the function so that any additional arguments in the future will also work)